src/test/java/com/epam/tc/hw9/test/TrelloApiTests.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.tc.hw9.test;
+
+import static com.epam.tc.hw9.service.TrelloServiceObject.correctResponseSpecification;
+import static com.epam.tc.hw9.service.TrelloServiceObject.getBoardFromResponse;
+import static com.epam.tc.hw9.service.TrelloServiceObject.getCardFromResponse;
+import static com.epam.tc.hw9.service.TrelloServiceObject.getListFromResponse;
+import static com.epam.tc.hw9.service.TrelloServiceObject.requestBuilder;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+import com.epam.tc.hw9.constants.URI;
+import com.epam.tc.hw9.dto.BoardDto;
+import com.epam.tc.hw9.dto.CardDto;
+import com.epam.tc.hw9.dto.ListDto;
+import com.epam.tc.hw9.service.Utils;
+import io.restassured.http.Method;
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+public class TrelloApiTests {
+
+    private BoardDto newBoard;
+    private ListDto newList;
+    private CardDto newCard;
+
+    @Test
+    public void createABoardTest() {
+        newBoard = getBoardFromResponse(requestBuilder()
+            .setMethod(Method.POST)
+            .setName(Utils.BOARD_NAME)
+            .buildRequest().sendRequest(URI.BOARD_ENDPOINT)
+            .then().assertThat()
+            .spec(correctResponseSpecification())
+            .extract().response());
+
+        assertThat("Incorrect name of the Board", newBoard.getName().equals(Utils.BOARD_NAME));
```

Comment:
create separate class with asserts with method, which can be called like

trelloAsserts(newBoard).assertNameIsCorrect(name)